### PR TITLE
7.16 Release notes

### PR DESCRIPTION
**Pull request by lzdanski**
Relates to https://github.com/astronomer/updates.astronomer.io/pull/228

### CODE REVIEW

File: astro/release-notes.md [hunk 1]
```diff
@@ -25,6 +25,18 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
 
 <!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->
 
+## July 16, 2024
+
+### Additional improvements
+
+- Added the following metrics to the Unviersal Metrics Export:
```
Comment (vishwas-astro) [inline astro/release-notes.md:32]: typo in "Universal"

File: astro/release-notes.md [hunk 2]
```diff
@@ -25,6 +25,18 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+    - The new `airflow_executor_open_slots`, `airflow_dagrun_dependency-check`, and `airflow_dagrun_dependency-check.<dag_id>` metrics allow you to collect metrics about your DAGs and executor status. <!-- https://github.com/astronomer/astro/pull/22430-->
+    -  Monitor task execution with `kube_pod_container_resource_limits`. This new metric enables you to track resource use against the configured limits so you can understand if task execution meets your configured CPU, memory, or storage limits. <!--https://github.com/astronomer/astro/pull/22322-->
```
Comment (chomipi88) [inline astro/release-notes.md:34]: Only enabling for Celery Workers, KE and KPO pods at this time.

Comment (chomipi88) [reply astro/release-notes.md:34]: Nvm you have it as task execution which is correct.

**Review by chomipi88 — APPROVED** [review]

File: astro/release-notes.md [hunk 3]
```diff
@@ -25,6 +25,25 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+
+### Additional improvements
+
+- Added the following metrics to [Metrics Export](https://www.astronomer.io/docs/astro/export-metrics):
+    - The new `airflow_executor_open_slots`, `airflow_dagrun_dependency-check`, and `airflow_dagrun_dependency-check.<dag_id>` metrics allow you to collect metrics about your DAGs and executor status. <!-- https://github.com/astronomer/astro/pull/22430-->
+    -  Monitor task execution with `kube_pod_container_resource_limits`. This new metric enables you to track resource use against the configured limits so you can understand if task execution meets your configured CPU, memory, or storage limits. <!--https://github.com/astronomer/astro/pull/22322-->
```
Comment (lzdanski) [inline astro/release-notes.md:40]:
```suggestion
    -  Monitor task execution with `kube_pod_container_resource_limits`. This new metric enables you to track resource use against the configured limits so you can understand if task execution meets your configured CPU, memory, or storage limits for your Celery Workers or Kubernetes Executor and KubernetesPodOperator pods.
```

**Review by virajmparekh — APPROVED** [review]

File: astro/release-notes.md [hunk 4]
```diff
@@ -25,6 +25,19 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
 
 <!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->
 
+## July 16, 2024
+
+### Additional improvements
+
+- Added the following metrics to [Metrics Export](https://www.astronomer.io/docs/astro/export-metrics):
+    - The new `airflow_executor_open_slots`, `airflow_dagrun_dependency-check`, and `airflow_dagrun_dependency-check.<dag_id>` metrics allow you to collect metrics about your DAGs and executor status.
+    -  Monitor task execution with `kube_pod_container_resource_limits`. This new metric enables you to track resource use against the configured limits so you can understand if task execution meets your configured CPU, memory, or storage limits for your Celery Workers or Kubernetes Executor and KubernetesPodOperator pods.
```
Comment (merobi-hub) [inline astro/release-notes.md:34]:
```suggestion
    - Monitor task execution with `kube_pod_container_resource_limits`. This new metric enables you to track resource use against the configured limits so you can understand if task execution meets your configured CPU, memory, or storage limits for your Celery Workers or Kubernetes Executor and KubernetesPodOperator pods.
```

**Review by merobi-hub — APPROVED** [review]

**Review by chomipi88 — APPROVED** [review]

**Comment by lzdanski** [comment]
Approved by technical reviewers and edited by content reviewers. Merging.

**Pull request merged**